packages/block-library/src/gallery/save.js

```diff
@@ -9,43 +9,48 @@ import { RichText } from '@wordpress/block-editor';
 import { defaultColumnsNumber } from './shared';
 
 export default function save( { attributes } ) {
-	const { images, columns = defaultColumnsNumber( attributes ), imageCrop, linkTo } = attributes;
+	const { images, columns = defaultColumnsNumber( attributes ), imageCrop, caption, linkTo } = attributes;
+
 	return (
-		<ul className={ `columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` } >
-			{ images.map( ( image ) => {
-				let href;
+		<figure>
+			<ul className={ `blocks-gallery-grid columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` }>
```

Comment:
Why add an extra class here? Why not write `wp-block-gallery ul` to select it?
Any reason the `is-cropped` class can't be moved to the parent element?

Reply:
It's always better to target the specific element styles are being applied to with a classname. Using `.className element` selectors will make things messy if we ever need to add another `element` with a different purpose to the component. I know there are quite a few existing instances of element selectors in this codebase, but I'd like to propose we move away from that. Ideally, every element that requires CSS should have its own classname; it'll save us a lot of grief in the long run.

No reason `is-cropped` can't move; I left it there along with the other gallery-related classes like `columns` and `align`, but now I see `align` at least should be applied to the outer wrapper so might as well move the whole lot.